tests/sqllogictests/suites/base/03_common/03_0028_copy_into_stage

```diff
@@ -37,4 +37,26 @@ statement ok
 drop stage test
 
 statement ok
-DROP DATABASE db1
\ No newline at end of file
+DROP STAGE IF EXISTS hello
+
+statement ok
+CREATE STAGE IF NOT EXISTS hello
+
+statement ok
+COPY INTO @hello from (select number from numbers(1)) FILE_FORMAT = (type = parquet)
+
+statement ok
+CREATE TABLE word(c1 int , c2 int);
+
+statement error (?s)1006.* Fields in select statement is not equal with expected, select fields: 1, insert fields: 2
```

Comment:
```suggestion
statement error (?s)1006.* Number of columns in select list (1) does not match that of the corresponding table (2)"
```

Reply:
done

Reply:
```rust
Test finished, fail fast enabled, 1 out of 4407 records failed to run
0: statement is expected to fail with error:
	(?s)1006.* Number of columns in select list (1) does not match that of the corresponding table (2)
but got error:
Error: SelfError("sqllogictest failed")
	mysql client error: Server error: `ERROR HY000 (1105): BadArguments. Code: 1006, Text = Number of columns in select list (1) does not match that of the corresponding table (2).'
[SQL] COPY INTO world FROM (select number  from @hello )   file_format = (type = PARQUET);
at tests/sqllogictests/suites/base/03_common/03_0028_copy_into_stage:51
``` 
confused

Reply:
let me try `statement` -> `query`

Reply:
I got it, it`s regex, need to escape `(` and `)`